### ISSUE TITLE
如何忽略多余的标签

### ISSUE THREAD

**Issue by luhairong11** [foreverYoungGitHub/MTCNN#7]
训练人脸检测的时候是3部分训练数据，2个任务进行的，非人脸，人脸，部分人脸，分类标签分别为0，1，-1，请问如何忽略掉-1的分类标签进行分类训练呢，又是如何忽略掉非人脸的回归标签进行回归任务训练的呢，

**Comment by foreverYoungGitHub** [comment]
Good question. I'll answer the detail in the update readme.

Basically, when I train the face detection, the label of face and part of face is 1, while the label of non-face is 0. The value of regression box in the non-face is {0, 0, 0, 0}.

**Comment by luhairong11** [comment]
Thank you very much, but the author of the paper with three parts of the data for training, and you only 2 part of the data training, then your training data ratio is how much, non-face: part of face: face =?
